src/shared/game/ship_data.c

```
@@ -130,72 +159,66 @@ static void load_serialization_list(bc_ship_class_t *ship, const json_value_t *a
-        int flat_idx = ename ? find_subsys_by_name(ship, ename) : -1;
-        if (flat_idx >= 0) {
-            e->hp_index = flat_idx;
-            /* Use the flat subsystem's max_condition as the canonical value.
-             * HP is initialized from the flat array (bc_ship_init), so the
-             * serialization entry must use the same denominator for
-             * encode_condition() and power_tick condition_pct to be correct.
-             * Without this, ships where subsystems.json and serialization.json
-             * disagree on max_condition (e.g. Galor warp core: 3200 vs 5000)
-             * spawn with condition_pct < 1.0 and reduced/zero power. */
-            e->max_condition = ship->subsystems[flat_idx].max_condition;
+        int parent_hp = ename ? find_subsys_by_name(ship, ename) : -1;
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)